test/DGmethods/advection_diffusion/pseudo1D_advection_diffusion.jl

```diff
@@ -44,6 +44,7 @@ end
 function initial_condition!(::Pseudo1D{n, α, β, μ, δ}, state, aux, x,
```

Comment:
Can you figure out the units of these quantities as well so that you don't need `value`?